recipe/meta.yaml

```
@@ -8,19 +8,34 @@ package:
 source:
   url: https://github.com/kazuho/{{ name }}/archive/v{{ version }}.tar.gz
   sha256: 056805ca2691798f5545935a14bb477f2e1d827c9fb862e6e449dbea22801c7d
+  patches:
+    - patches/0001-Add-CMake-config-file-package-to-install.patch
 
 build:
+  ignore_run_exports:
+    # package is header-only
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
   number: 0
```

This needs to be incremented to indicate a new build.

```suggestion
  number: 1
```